README.md

```diff
@@ -81,13 +81,20 @@ The workflow that the system should support is:
 ## Tasks
 ### 1. Forex data retrieval
 #### What we want
+The candidate should be able to create a Service that periodically fetches from the Forex API provided of the subscribed currencies and persist on database in order to have a register of how currencies's values change along time.
 #### What we will evaluate
-
+* The way the candidates organizes code and looks for a neat and optimized environment.
+* How the candidate models the domain in order to handle more types of data sources.
 
 ### 2. API endpoint for retrieving the fetched forex data
 #### What we want
+Using the provided architecture in the API, we hope the candidate to implement a new endpoint (`[GET] /api/currency/:code/history` _could be an option_) that retrieves the temporary information of each currency (based on the currency code) collected by the service that communicates with the Forex API.
```

Comment:
```suggestion
Implement an API endpoint to retrieve the historical data for currently followed currencies.

A candidate for this would be a `GET` endpoint along the lines of `/api/currency/:code/history` or
``/api/forex/history`.

We already provide you with a working API at `services/api` where you could add your endpoint.
```